waltz-ng/client/physical-flows/physical-flow-registration.html

```diff
@@ -81,6 +81,48 @@
 
         </div>
 
+
+        <div ng-if="ctrl.visibility.editor == '' && ctrl.similarFlows.length > 0">
+            <div class="row" ng-hide="ctrl.visibility.similarFlows">
+                <div class="col-md-offset-8 col-md-2">
+                    <strong class="text-warning">Similar flows exist!</strong>
+                    <button class="btn btn-warning"
+                            ng-click="ctrl.visibility.similarFlows = true">
+                        Show
+                    </button>
+                </div>
+            </div>
+
+            <div ng-if="ctrl.visibility.similarFlows">
+                <h4>
+                    Existing flows from
+                    <waltz-entity-icon-label entity-ref="ctrl.sourceEntity"></waltz-entity-icon-label>
+                    to
+                    <waltz-entity-icon-label entity-ref="ctrl.targetEntity"></waltz-entity-icon-label>
+                </h4>
+                <waltz-grid column-defs="ctrl.similarFlowDefs"
+                            row-data="ctrl.similarFlows"
+                            class="small">
+                </waltz-grid>
```

Comment:
Primarily to keep the styling between this and other areas where physical flows are in a table.  Can change to a <table> instead?